dumper/app/src/main/resources/oracle-stats/native/db-objects-synonym-public.sql

```diff
@@ -0,0 +1,50 @@
+-- Copyright 2022-2024 Google LLC
+-- Copyright 2013-2021 CompilerWorks
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+SELECT
```

Comment:
What is a downside of including this query in the `UNION ALL` statement in `db-objects-no-join` script?

Reply:
In this case there's the problem of `LIKE` predicates, which can mess with optimizations. Logically, they should have no effect on unrelated parts of the query, but in practice they do occasionally cause the database to plan things badly.

In general, I think breaking up particularly difficult queries into smaller parts makes our job easier. It's difficult to understand what happens in those queries as well as the original authors do, so it's much safer if we keep things as simple as possible.